### ISSUE TITLE
Test Sagepay tokens being used for payment sources

### ISSUE THREAD

**Issue by lukeholder** [craftcms/commerce-sagepay#11]
https://github.com/pixelandtonic/Commerce/issues/21

Make `supportsPaymentSources` true by and implement:
https://github.com/pixelandtonic/commerce-sagepay/commit/5aed8a888ec6e4bb2099bc82cb0b854eedad46cd

**Comment by terryupton** [comment]
@lukeholder and @andris-sevcenko - I am working with a client who wants to add this facility and have now paid to enable tokens on their SagePay account. They would be happy to give you access to the test account and I can help set this up. Would this be something you would be use to try and get this working and ability to save cards with the SagePay gateway plugin?

https://github.com/craftcms/commerce-sagepay/issues/13

**Comment by terryupton** [comment]
Hey @lukeholder and @andris-sevcenko - hope you guys are doing ok.
I just wondered if there was any progress on the token stuff (saving cards) for SagePay or any ETA/Roadmap for this at all?
Thanks

**Comment by terryupton** [comment]
Hey Guys.

Any plans to integrate this still into this plugin?
We currently extend this plugin, but it would be much easier if we could have it native in the gatweay.
Thanks.

**Comment by lukeholder** [comment]
@terryupton If you have already done the work a PR is welcome!

**Comment by terryupton** [comment]
> [@terryupton](https://github.com/terryupton) If you have already done the work a PR is welcome!

@lukeholder we had this working on Commerce 3, but we decided to drop our custom gateway on commerce 4 and revert back to this version. Only because of the additional development required.

**Comment by lukeholder** [comment]
No worries. So we can leave this feature request open then?

**Comment by terryupton** [comment]
Yes please. It would be great if there was some official support for this.
